examples/hello-cage/README.md

```diff
@@ -0,0 +1,90 @@
+# HELLO CAGE
+
+A simple node server that can be deployed into a cage. 
+
+## Endpoints
+
+### Hello
+
+A simple endpoint that return a hello message and echoes any body passed into it back in the response. Any evervault encrypted strings will be decrypted in the response to demonstrate automatic decryption within cages.
```

Comment:
Should we capitalise evervault and cage everywhere?

Reply:
Good shout, will do